.stylelintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
       },
     ],
     'color-function-notation': 'legacy',
+    'color-function-alias-notation': 'legacy',
```

Comment:
I had to add this to stop the rule that is included in what we extend (`stylelint-config-standard`) from fighting with the `color-function-notation` one we add above. Agreed with @rcantin-w that we could/should maybe try to remove _both_ of these rules in a separate PR to go modern all the way, but this was the smallest change required to make the commit hook work